prime-router/metadata/fhir_mapping/hl7/segments/SPM/Specimen.yml

```diff
@@ -0,0 +1,222 @@
+resourceType: Specimen
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+identifier:
+  generateList: true
+  expressionType: nested
+  expressions:
+    - condition: $spm21 NOT_NULL
+      valueOf: datatypes/EIP/Identifier-PlacerAssignedIdentifier
+      expressionType: resource
+      specs: SPM.2
+      vars:
+        spm21: SPM.2.1
+      constants:
+        hl7Use: "specimen-id-placer"
+        backupExtension: "filler-assigned-identifier"
+    - condition: $spm22 NOT_NULL
+      valueOf: datatypes/EIP/Identifier-FillerAssignedIdentifier
+      expressionType: resource
+      specs: SPM.2
+      vars:
+        spm22: SPM.2.2
+      constants:
+        hl7Use: "specimen-id-filler"
+        backupExtension: "placer-assigned-identifier"
+    - valueOf: datatypes/CX/Identifier
+      expressionType: resource
+      generateList: true
+      specs: SPM.31 *
+      constants:
+        identifierIndexName: "other-specimen-id"
+    - condition: $spm32 NOT_NULL
+      valueOf: datatypes/EI/Identifier
+      expressionType: resource
+      specs: SPM.32
+      vars:
+        spm32: SPM.32
+      constants:
+        hl7Use: "shipment-id"
+        code: "SHIP"
+        system: "http://terminology.hl7.org/CodeSystem/v2-0203"
+
+type:
+  valueOf: datatypes/CWE/CodeableConcept
+  expressionType: resource
+  specs: SPM.4
+
+container:
+  condition: $spm6 NOT_NULL || $spm27 NOT_NULL
+  expressionType: nested
+  generateList: true
+  vars:
+    spm6: SPM.6
+    spm27: SPM.27
+  expressionsMap:
+    additiveCodeableConcept:
+      valueOf: datatypes/CWE/CodeableConcept
+      expressionType: resource
+      specs: SPM.6 *
+    type:
+      valueOf: datatypes/CWE/CodeableConcept
+      expressionType: resource
+      specs: SPM.27
+
+collection:
+  condition: $spm7 NOT_NULL || $spm8 NOT_NULL || $spm9 NOT_NULL || $spm12 NOT_NULL
+  expressionType: nested
+  generateList: true
+  vars:
+    spm7: STRING, SPM.7
```

Comment:
Maybe?
```suggestion
    spm7: STRING_ALL, SPM.7
```

Reply:
ran some tests and it should be either `STRING_ALL` or no type. So I opted for no type